OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3707,66 +3710,93 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         on <locus target="#1r"/>. Another sticker with the signature ‘Ms 5’ printed on
                                         it is glued below it.</desc>
                                 </item>
-                                
+
                                 <item xml:id="e10">
                                     <locus target="#1r"/>
                                     <desc>The title of the manuscript in transliteration is written in pencil.</desc>
                                     <q xml:lang="gez">Ta'amara Maryam</q>
                                 </item>
-                                
+
                                 <item xml:id="e11">
                                     <locus target="#123r"/>
                                     <desc>Written in pencil.</desc>
                                     <q xml:lang="en">33118</q>
                                 </item>
-                                
+
                                 <item xml:id="e12">
                                     <locus target="#1r #44v #69v #96v #123v"/>
                                     <desc>Stamps.</desc> <!--I cannot read the content of the stamps-->
                                 </item>
                             </list>
                         </additions>
-                        
+
                         <bindingDesc>
                             <binding xml:id="binding" contemporary="true">
-                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with reddish-brown blind-tooled <material
-                                        key="leather">leather</material>. The front and back board are decorated with red velvet inlays.</decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                                    <material key="wood"/>
-                                    <material key="leather"/>
-                                    <material key="textile"/>
+                                    <decoNote xml:id="b1" type="bindingMaterial">
+                                      The book is bound in <material key="wood">wooden</material> boards covered with reddish-brown blind-tooled <material
```

Comment:
above, "tooled leather" - is this a difference or an inconsistency?

Reply:
tooling is the parent term of blind tooling and gold tooling. So it is less accurate. I changed it!